DESCRIPTION

```diff
@@ -1,5 +1,5 @@
 Package: BoutrosLab.plotting.general
-Version: 7.0.5
+Version: 7.0.6
```

Comment:
v7.0.5 isn't released yet, so we can just add this patch without incrementing the version.